packages/clerk-js/src/ui/elements/SegmentedControl.tsx

```diff
@@ -120,10 +125,10 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(({ text, value }
             isActive={isSelected}
             sx={t => ({
               position: 'relative',
-              padding: `${t.space.$1} ${t.space.$2x5}`,
+              width: variant === 'large' ? '100%' : 'auto',
```

Comment:
Can we add a fullWidth prop to control the layout vs inferring from large size. We could want a large size implementation that is not full width.